### DISCUSSION TITLE
Help!

### DISCUSSION THREAD

**Discussion by timmckai1** [Koenkk/zigbee2mqtt#10518] [General]
Hello, I'm kinda new to this. I've been using Zha with home assistant fine for a while. I want to change over to ZigBee2mqtt but having issues. I have it set up on an unraid docker. When I start the container it seems to work fine, I can login to the front-end. But only for about 1min then the it just stops.  The only error I can see is: Error: Unrecognized nodeId:4739. 

I have no idea what this is?

**Comment by mausch** [comment]
Hi, you'll probably need the stack trace of that error to start diagnosing things.
If you search for "unrecognised nodeid" here on github you'll see many similar errors and their stack traces.
